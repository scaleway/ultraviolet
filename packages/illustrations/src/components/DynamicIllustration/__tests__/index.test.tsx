import {
  consoleDarkerTheme,
  consoleDarkTheme,
  consoleLightTheme,
} from '@ultraviolet/themes'
import { shouldMatchEmotionSnapshot } from '@utils/test'
import { describe, it } from 'vitest'
import { DynamicIllustration } from '..'

describe('dynamicIllustration', () => {
  it('should work with consoleLightTheme', () =>
    shouldMatchEmotionSnapshot(
      <DynamicIllustration name="empty" />,
      consoleLightTheme,
    ))
  it('should work with consoleDarkTheme', () =>
    shouldMatchEmotionSnapshot(
      <DynamicIllustration name="empty" />,
      consoleDarkTheme,
    ))

  it('should work with consoleDarkerTheme', () =>
    shouldMatchEmotionSnapshot(
      <DynamicIllustration name="empty" />,
      consoleDarkerTheme,
    ))
})
