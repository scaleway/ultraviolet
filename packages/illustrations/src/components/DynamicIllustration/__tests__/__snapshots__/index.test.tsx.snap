// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`dynamicIllustration > should work with consoleDarkTheme 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <img
      alt="empty"
      src="https://assets.scaleway.com/illustrations/various/empty/empty-light.webp"
    />
  </div>
</DocumentFragment>
`;

exports[`dynamicIllustration > should work with consoleDarkerTheme 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <img
      alt="empty"
      src="https://assets.scaleway.com/illustrations/various/empty/empty-light.webp"
    />
  </div>
</DocumentFragment>
`;

exports[`dynamicIllustration > should work with consoleLightTheme 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <img
      alt="empty"
      src="https://assets.scaleway.com/illustrations/various/empty/empty-light.webp"
    />
  </div>
</DocumentFragment>
`;
